### PR TITLE
Rename package to MetopDatasets.jl

### PR DESCRIPTION
**Pull request by lupemba**
- Rename package to MetopDatasets.jl
- Use new UUID to identify the package
- Remove obsolete gitlab CI scripts and links
- Add a bit of info about the renaming to README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,10 @@
...
+This package is still in the early development phase. Note that the package was previously name **MetopNative.jl** and was hosted on the [EUMETSAT GitLab](https://gitlab.eumetsat.int/eumetlab/cross-cutting-tools/MetopNative.jl) 
+# MetopDatasets.jl
 
-MetopNative.jl is a package for reading products from the [METOP satellites](https://www.eumetsat.int/our-satellites/metop-series) using the native binary format specified for each product. The METOP satellites are part of the EUMETSAT-POLAR-SYSTEM (EPS) and produces long series of near-realtime weather and climate observation. Learn more and access the products on [EUMETSATs user-portal](https://user.eumetsat.int/dashboard).
+MetopDatasets.jl is a package for reading products from the [METOP satellites](https://www.eumetsat.int/our-satellites/metop-series) using the native binary format specified for each product. The METOP satellites are part of the EUMETSAT-POLAR-SYSTEM (EPS) and produces long series of near-realtime weather and climate observation. Learn more and access the products on [EUMETSATs user-portal](https://user.eumetsat.int/dashboard).
```
Comment (wilzewski) [inline README.md:5]: produces -> produce
near-realtime -> near-real-time (?)

File: README.md [hunk 2]
```diff
@@ -1,12 +1,10 @@
...
 
-MetopNative.jl has two API's for reading native products. The first API is `MetopDataset` that implements the [CommonDataModel.jl](https://github.com/JuliaGeo/CommonDataModel.jl) interface and thus provides data access similar to e.g. [NCDatasets.jl](https://github.com/Alexander-Barth/NCDatasets.jl) and [GRIBDatasets.jl](https://github.com/JuliaGeo/GRIBDatasets.jl). The second API is `MetopProduct` which reads an entire product into a Julia object with a data structure similar to the native file structure. 
+MetopDatasets.jl has two API's for reading native products. The first API is `MetopDataset` that implements the [CommonDataModel.jl](https://github.com/JuliaGeo/CommonDataModel.jl) interface and thus provides data access similar to e.g. [NCDatasets.jl](https://github.com/Alexander-Barth/NCDatasets.jl) and [GRIBDatasets.jl](https://github.com/JuliaGeo/GRIBDatasets.jl). The second API is `MetopProduct` which reads an entire product into a Julia object with a data structure similar to the native file structure.
```
Comment (wilzewski) [inline README.md:7]: API's -> APIs

**Review by wilzewski — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (lupemba) [reply README.md]: https://www.eumetsat.int/real-time-imagery/earth-view and https://en.wikipedia.org/wiki/Real-time_computing#:~:text=The%20term%20%22near%20real%2Dtime,or%20feedback%20and%20control%20purposes. say "near real-time" so I will go with that.

File: README.md (hunk 2 above)
Comment (lupemba) [reply README.md]: Maybe it would be good to remove `MetopProduct` in the future. It is not really needed and the package would be simpler without.

**Pull request merged**